src/app/components/BibPage/ElectronicResources.jsx

```diff
@@ -1,22 +1,45 @@
-import { Card, CardContent, CardHeading } from '@nypl/design-system-react-components'
+import { Card, CardContent, CardHeading, Button, Icon } from '@nypl/design-system-react-components'
 import generateERLinksList from '../../utils/electronicResources'
-import React from 'react';
+import React, { useRef, useState } from 'react';
 
 /**
  * ElectronicResources renders a list of electronic resources links, sans aeon links
  * @param {array} electronicResources - an array of electronic resources, passed in as a prop from the BibPage component
+ * @param {boolean} isTestMode - a boolean value only used while running unit tests on this component
  */
-const ElectronicResources = ({ electronicResources }) => {
+
+
+const ElectronicResources = ({ electronicResources, isTestMode = false }) => {
+  const [showMore, setShowMore] = useState(true)
+  const scrollToRef = useRef();
+
   if (!electronicResources || !electronicResources.length) {
     return null;
   }
-  const resources = generateERLinksList(electronicResources)
-  return (<Card isBordered padding="16px">
+
+  const allResources = generateERLinksList(electronicResources)
+  const threeResources = generateERLinksList(electronicResources.slice(0, 3))
+  const more = `all ${electronicResources.length}`
+  const less = 'fewer'
+
+  const onClick = () => {
+    if (!isTestMode) scrollToRef.current.scrollIntoView({ behavior: 'smooth' })
+    setShowMore((prev) => {
+      return !prev
+    })
+  }
+
+  return (<Card ref={scrollToRef} isBordered padding="16px">
     <CardHeading level="three" id="no-img1-heading1">
       Available Online
     </CardHeading>
     <CardContent>
-      {resources}
+      {showMore ? threeResources : allResources}
+      {electronicResources.length > 3 ?
+        <Button style={{ textDecoration: 'none' }} isBordered='false' id='see-more-button' onClick={onClick} buttonType='link'>
```

Comment:
A few notes about this:
- `isBordered` is not a valid prop for the `Button` component,
-  Update the `style` attribute to use the prop `textDecoration="none"`,
- likewise, add `border="none"`

```
<Button textDecoration="none" border="none" ....>...`
```

Reply:
oh sweet! got the border to go away.